disco/settings.py

```diff
@@ -441,60 +462,86 @@
         "iconclass": "fa fa-bolt",
         "component": "views/components/cards/file-renderers/xy-reader",
         "ext": "txt",
-        "type": "text/plain",   
+        "type": "text/plain",
         "exclude": "",
     },
 ]
 
 X_FRAME_OPTIONS = "SAMEORIGIN"
 
 FORMATS = [
```

Comment:
This array isn't needed anymore per https://github.com/archesproject/arches-for-science/issues/1380

But shouldn't cause a problem.